parts/linux/cloud-init/artifacts/setup-custom-search-domains.sh

```diff
@@ -6,6 +6,6 @@ source "${CSE_DISTRO_HELPERS_FILEPATH}"
 echo "  dns-search ${CUSTOM_SEARCH_DOMAIN_NAME}" | tee -a /etc/network/interfaces.d/50-cloud-init.cfg
 systemctl_restart 20 5 10 networking
 wait_for_apt_locks
-retrycmd_if_failure 10 5 120 apt-get -y install realmd sssd sssd-tools samba-common samba samba-common python2.7 samba-libs packagekit
+retrycmd_if_failure 10 5 120 apt-get -y install realmd sssd sssd-tools samba-common samba samba-common python2.7 samba-libs packagekit || exit 1
```

Comment:
can we add a comment clarifying that the `exit 1` will be trapped by `exit $ERR_CUSTOM_SEARCH_DOMAINS_FAIL` within CSE?

Reply:
I could be wrong, but I think we dont support custom search domain anymore, this is legacy code carried over from aks-engine. it isnt in the ManagedCluster data model as well

Reply:
Ah okay, will leave this as it is for now then, we could remove the whole thing later on.